packages/flutter/lib/src/services/asset_bundle.dart

```diff
@@ -102,9 +102,9 @@ abstract class AssetBundle {
 class NetworkAssetBundle extends AssetBundle {
   /// Creates an network asset bundle that resolves asset keys as URLs relative
   /// to the given base URL.
-  NetworkAssetBundle(Uri baseUrl)
+  NetworkAssetBundle(Uri baseUrl, [HttpClient httpClient])
```

Comment:
revert this line. It appears there was a merge conflict.

Reply:
actually this was intended to be able to inject a mock http client to make the test works, but on second thought we can do this without injecting / creating mock http client since it will always returns 404 anyway